examples/autoinstrument/deployment.yaml

```
@@ -29,14 +29,20 @@ spec:
     spec:
       containers:
         - name: hello-autoinstrument-java
-          image: gcr.io/%GOOGLE_CLOUD_PROJECT%/hello-autoinstrument-java:latest
+          image: us-central1-docker.pkg.dev/%GOOGLE_CLOUD_PROJECT%/opentelemetry-sample-apps/hello-autoinstrument-java:latest
           # required for resource detection in GKE environment
           env:
-            - name: NAMESPACE
+            - name: GOOGLE_CLOUD_PROJECT
+              value: %GOOGLE_CLOUD_PROJECT%
+            - name: POD_NAME
               valueFrom:
                 fieldRef:
-                  fieldPath: metadata.namespace
-            - name: CONTAINER_NAME
+                  fieldPath: metadata.name
+            - name: NAMESPACE_NAME
               valueFrom:
                 fieldRef:
-                  fieldPath: metadata.name
+                  fieldPath: metadata.namespace
+            - name: CONTAINER_NAME
+              value: hello-autoinstrument-java-container
```

```suggestion
              value: hello-autoinstrument-java
```
This should match the container name